command/loadbot/loadbot_command.go

```diff
@@ -372,9 +374,14 @@ type LoadbotResult struct {
 	TurnAroundData    TxnTurnAroundData    `json:"turnAroundData"`
 	BlockData         TxnBlockData         `json:"blockData"`
 	DetailedErrorData TxnDetailedErrorData `json:"detailedErrorData,omitempty"`
+	RealTPS           uint64               `json:"realTps"`
 }
 
 func (lr *LoadbotResult) extractExecutionData(metrics *Metrics) {
+	// calculate real transactions per second value
+	// by deviding total transactions by total time in seconds
+	//nolint:lll
```

Comment:
Please remove the nolint clause and break up the next line into several lines for readability, you can do it in Go

Reply:
Removed linter clause and broke that line of code to multiple lines f9886aed1bef4a43315d3b30f970cda6e1c9e8c2